app/javascript/packs/application.js

```diff
@@ -16,50 +16,27 @@ ActiveStorage.start()
 
 //= require_tree.
 
-const themeDarkGrey = "#474B4F";
-const themeDarkBlue = "#3A546C";
-const themeBackgroundColor = "#F0F0F0";
-const whiteColor = "#FFF";
-
-function requestEventPassword(eventId, eventName) {
-    console.log(eventId);
-    console.log(eventName);
-}
-
-function addShortcutKeys() {
```

Comment:
Was this function ever used / was it deleted here intentionally?

Reply:
i don’t think it was ever used. it only ever had console logs in it anyway

Reply:
All good then haha